earthmover/earthmover.py

```diff
@@ -379,7 +378,7 @@ def generate(self, selector):
                         )
                         self.do_generate = False
 
-        elif 'state_file' not in self.state_configs:
+        elif not self.state_configs.get('state_file', False):

```

Comment:
Technically, the default value `None` and the supplied `False` will result in the same output.